README.md

```diff
@@ -11,28 +11,78 @@ We use the GitHub issue tracker only for issues related to the development of Ub
 
 ## Configuration
 
-The Flutter UI can be configured using a [YAML](https://yaml.org/) file.
-  ```yaml
-  theme:
-    accent-color: "#ff0000"
-    elevated-button-color: "#00ff00"
-    elevated-button-text-color: "#0000ff"
-    window-title: "Custom Window Title"
-
-  pages:
-    locale:
-      image: "/path/to/image.png"
-      visible: true
-    ...
-  ```
-
-`visible` defaults to true if not specified and the `image` defaults to an image in in `assets`
-directory, if there is one specified for the page.
-
-Lookup order:
-- `/etc/whitelabel.{yaml,yml}` (admin)
-- `/usr/local/share/whitelabel.{yaml,yml}` (oem)
-- `/usr/share/whitelabel.{yaml,yml}` (distro)
+The full configuration guide can be read [here](https://docs.google.com/document/d/10R0YOj4e8BTv6XPw9OE_y7GDy72xPqA5XP5lu0M7VbE/edit?usp=sharing).
+
+But the basics are that the UI can be configured using a [YAML](https://yaml.org/) file that is read from here:
+`/usr/share/desktop-provision/whitelabel.yaml`
+
+The configuration file has the following structure:
+
+```yaml
+# (Optional) Drives overall behaviour for specific provisioners.
+#
+# Options:
+# - standard (default): the common provision flow for Ubuntu Desktop and Flavours
+# - oem: enables the eula page and disables the user creation page during bootstrap
+mode: standard | oem
+
+# (Optional) When set, the light and dark theme is inherited from ubuntu-flutter-plugins and the distro name is set.
+#
+# Options:
+# - budgie
+# - cinnamon
+# - edubuntu
+# - kubuntu
+# - kylin
+# - lubuntu
+# - mate
+# - studio
+# - unity
+# - xubuntu
+flavour: <name>
+
+# (Optional) Sets the window's title (e.g. the text in alt|super + tab)
+app-name: <string>
+
+# (Optional) Overrides the theme's accent colors
+theme:
+  light:
+    accent-color: <color-hex-code> # i.e. #ff0011
```

Comment:
We should clarify/emphasize that the color code needs to be a string.
```yaml
accent-color: #ff0000
```
won't work as `#` denotes a comment, only
```yaml
accent-color: "#ff0000"
```
will.

Reply:
Ah, copy paste mistake from the google doc, updated both now.